frontend/src/components/common/form/RadioGroup.tsx

```diff
@@ -39,6 +39,8 @@ type OptionalAttributes = {
   toolTip?: string;
   /** id for tooltip */
   toolTipId?: string;
+  /** Flex direction for the group */
+  flexDirection?: string;
```

Comment:
nit: a bit better typing for this would be:
```tsx
flexDirection?: 'column' | 'row'
```

That way the TS compiler would complain if you ever assign `flexDirection: 'cat'`